content/en/logs/explorer/patterns.md

```diff
@@ -33,17 +33,16 @@ The patterns view is helpful for detecting and filtering noisy error patterns th
 
 Switch to the patterns view to automatically see your log patterns for the selected context. A [context][1] is composed of a time range and a search query.
 
-**Note**: The pattern detection is based on 10,000 log samples. Refine the search to see patterns limited to a specific subset of logs:
-
-{{< img src="logs/explorer/log_patterns.png" alt="Log Patterns" responsive="true" style="width:90%;">}}
+Each pattern comes with highlights, to get you straight to its characteristic features:
 
-Sections of the logs that vary within the pattern are highlighted to help you quickly identify differences across log lines.
+* Sections of the logs that vary within the pattern are highlighted, to help you quickly identify differences across log lines. 
+* A minigraph displays a rough timeline for the volume of its logs, to help you identify how that pattern differs from other patterns. 
```

Comment:
have we decided for sure to call this "minigraph" - "mini graph" might make more sense

Reply:
all good for me

Reply:
let's make it mini graph instead.